### PR TITLE
SSCSCI-1397

### PR DESCRIPTION
**Pull request by tomxelliott**
### Jira link
https://tools.hmcts.net/jira/browse/SSCSCI-1397

### Change description
Add English IBCA dynamic content for documents

### CODE REVIEW

**Review by linusnorton — DISMISSED** [review]

File: src/main/java/uk/gov/hmcts/reform/sscs/evidenceshare/service/placeholders/PlaceholderConstants.java [hunk 1]
```diff
@@ -6,13 +6,16 @@ public final class PlaceholderConstants {
...
+    public static final String SHOULD_HIDE_IBCA_REFERENCE = "should_hide_ibca_reference";
     public static final String APPELLANT_FULL_NAME_LITERAL = "appellant_full_name";
     public static final String CASE_ID_LITERAL = "case_id";
     public static final String NINO_LITERAL = "nino";
+    public static final String IBCA_REFERENCE_LITERAL = "ibca_reference";
     public static final String GENERATED_DATE_LITERAL = "generated_date";
     public static final String WELSH_GENERATED_DATE_LITERAL = "welsh_generated_date";
     public static final String SSCS_URL_LITERAL = "sscs_url";
     public static final String SSCS_URL = "https://www.gov.uk/appeal-benefit-decision";
+    public static final String IBCA_URL = "www.gov.uk/guidance/appeal-an-infected-blood-compensation-decision";
```
Comment (nilay913) [inline src/main/java/uk/gov/hmcts/reform/sscs/evidenceshare/service/placeholders/PlaceholderConstants.java:18]: not need the https:// ?

Comment (tomxelliott) [reply src/main/java/uk/gov/hmcts/reform/sscs/evidenceshare/service/placeholders/PlaceholderConstants.java:18]: Yeah good shout. I just took the value from the ticket. Let me look into this

**Review by nilay913 — APPROVED** [review]

**Review by linusnorton — APPROVED** [review]

**Review by nilay913 — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/sscs/tyanotifications/personalisation/Personalisation.java [hunk 2]
```diff
@@ -558,8 +589,13 @@ Map<String, Object> setEvidenceProcessingAddress(Map<String, Object> personalisa
             personalisation.put(REGIONAL_OFFICE_POSTCODE_LITERAL, rpc.getPostcode());
         }
 
-        personalisation.put(PHONE_NUMBER_WELSH, evidenceProperties.getAddress().getTelephoneWelsh());
-        personalisation.put(PHONE_NUMBER, determinePhoneNumber(rpc));
+        if (!Objects.equals(ccdResponse.getIsScottishCase(), "Yes") && ccdResponse.isIbcCase()) {
```
Comment (linusnorton) [inline src/main/java/uk/gov/hmcts/reform/sscs/tyanotifications/personalisation/Personalisation.java:592]: `"Yes".equals(ccdResponse.getIsScottishCase())` don't we have a helper function for this already?

**Review by linusnorton — DISMISSED** [review]

**Review by nilay913 — DISMISSED** [review]

File: src/main/java/uk/gov/hmcts/reform/sscs/tyanotifications/personalisation/Personalisation.java [hunk 3]
```diff
@@ -558,8 +589,13 @@ Map<String, Object> setEvidenceProcessingAddress(Map<String, Object> personalisa
...
+            personalisation.put(PHONE_NUMBER_WELSH, "03001234567");
```
Comment (tomxelliott) [inline src/main/java/uk/gov/hmcts/reform/sscs/tyanotifications/personalisation/Personalisation.java:593]: Update this value to be `0300 303 5170` as per new requirement on SSCSCI-1401

**Review by nilay913 — DISMISSED** [review]

File: src/main/java/uk/gov/hmcts/reform/sscs/tyanotifications/config/AppConstants.java [hunk 4]
```diff
@@ -32,6 +32,18 @@ public final class AppConstants {
     public static final String ZONE_ID = "Europe/London";
     public static final int MAX_DWP_RESPONSE_DAYS = 28;
     public static final int MAX_DWP_RESPONSE_DAYS_CHILD_SUPPORT = 42;
+    public static final String IBC_ACRONYM = "IBC";
+    public static final String IBC_ACRONYM_WELSH = "IGH";
+    public static final String IBCA_ACRONYM = "IBCA";
```
Comment (nilay913) [inline src/main/java/uk/gov/hmcts/reform/sscs/tyanotifications/config/AppConstants.java:37]: this is a duplicate of line 22

**Review by Tim97eng — APPROVED** [review]
Looks good to me!

**Pull request merged**